packages/terra-switch/src/terra-dev-site/test/switch/DeafultSwitch.test.jsx

```diff
@@ -0,0 +1,20 @@
+import React, { useState } from 'react';
```

Comment:
correct the file name DefaultSwitch

Reply:
Oops my bad..! corrected : 82255613e7490a31855ffcb6114fce680a8934c6